src/main/java/com/sda/microblogging/entity/DTO/user/UserSavedDTO.java

```diff
@@ -0,0 +1,19 @@
+package com.sda.microblogging.entity.DTO.user;
+
+import com.sda.microblogging.entity.Role;
+import lombok.*;
+
+import java.sql.Date;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class UserSavedDTO {
+    private Integer userId;
+    private String username;
+    private String email;
+    private Date creationDate;
+    private Role role;
+}
```

Comment:
How do you save a user without password?

Reply:
this DTO - is in just output, actually I guess the most important is userId in it: proves that user was saved already and some id was assigned to it, the rest fields are not much important..
And as input for save - is UserSignUpDTO

e.g. public ResponseEntity<UserSavedDTO> signUp(@Valid @RequestBody UserSignUpDTO userSignUpDTO)